mesh.zone

```diff
@@ -18,6 +18,7 @@ jim	A 10.70.140.70
 mail	A 10.70.140.70
 donuts A 10.70.73.29
 unifi A 10.70.95.63
+netbox A 10.70.95.135
```

Comment:
`10.70.90.135` not `10.70.95.135` - `10.70.95.0/24` is sn3 oob.

Reply:
Fixed in 514c37c